### PR TITLE
Updating GitHub action versions to the latest TSCCR approved version

### PR DESCRIPTION
**Pull request by NicoletaPopoviciu**
### :+1: Definition of Done

- [ ] New functionality works?
- [ ] Tests added?
- [ ] Docs updated?

### :speech_balloon: Using the [Netlify feedback ladder](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify/)

- :mount_fuji: **[mountain]**: Blocking and requires immediate action.
- :moyai: **[boulder]**: Blocking.
- :white_circle: **[pebble]**: Non-blocking, but requires future action.
- :hourglass_flowing_sand: **[sand]**: Non-blocking, but requires future consideration.
- :sparkles: **[dust]**: Non-blocking. "Take it or leave it"

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,10 +28,10 @@ jobs:
...
 
     steps:
-      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
 
       - name: Install Node
-        uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3.7.0
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
         with:
           node-version: 18.x
```
Comment (WenInCode) [inline .github/workflows/ci.yml:36]: Is the plan to have a separate PR for switching to node 20?

**Review by WenInCode — APPROVED** [review]
TSCCR gha update looks good, thanks @NicoletaPopoviciu !

Left one question for you but LGTM overall

File: .github/workflows/ci.yml (hunk 1 above)
Comment (NicoletaPopoviciu) [reply .github/workflows/ci.yml:36]: Updating GitHub action versions to the latest TSCCR approved version addresses Node16 deprecations

**Comment by NicoletaPopoviciu** [comment]
@WenInCode please merge when you get a chance;

**Pull request merged**